v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/SpannerChangeStreamsToGcsOptions.java

```diff
@@ -103,10 +97,7 @@ public interface SpannerChangeStreamsToGcsOptions
       optional = true,
       description = "Cloud Spanner metadata table name",
       helpText =
-          "The Cloud Spanner change streams connector metadata table name to use. If not provided,"
-              + " a Cloud Spanner change streams connector metadata table will automatically be"
-              + " created during the pipeline flow. This parameter must be provided when updating"
-              + " an existing pipeline and should not be provided otherwise.")
+          "The Spanner change streams connector metadata table name to use. If not provided, a Spanner change streams metadata table is automatically created during the pipeline flow. You must provide a value for this parameter when updating an existing pipeline. Otherwise, don't use this parameter.")
```

Comment:
```suggestion
          "The Spanner change streams connector metadata table name to use. If not provided, a Spanner change streams metadata table is automatically created during pipeline execution. You must provide a value for this parameter when updating an existing pipeline. Otherwise, don't use this parameter.")
```